aspnetcore/tutorials/index.md

```diff
@@ -59,3 +60,7 @@ The following step-by-step guides for developing ASP.NET Core applications are a
 * [ASP.NET Core on Nano Server](nano-server.md)
 * [ASP.NET Core and Azure Service Fabric](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-add-a-web-frontend)
 
+## Downloading a sample
```

Comment:
How to download a sample

Also could add `<a id="download"></a>` to make it a little easier to link directly to this section and insure against links breaking if we ever change the h2 heading.